tests/annotations/tests.py

```diff
@@ -1167,6 +1170,27 @@ def test_alias_forbidden_chars(self):
                 with self.assertRaisesMessage(ValueError, msg):
                     Book.objects.annotate(**{crafted_alias: Value(1)})
 
+    @skipUnless(connection.vendor == "postgresql", "PostgreSQL tests")
+    def test_set_returning_functions(self):
+        """Test that utilizing the set returning flag returns the expected count"""
```

Comment:
```suggestion
    @skipUnlessDBFeature("supports_json_field")
    def test_set_returning_functions(self):
```

Reply:
@sarahboyce I think this is causing an issue since `jsonb_path_query` is a postgres-specific (to my understanding) function. So even if other databases support JSONField, they won't necessarily support the `jsonb_path_query`. Would it make sense to revert to the initial `skipUnless` or do you prefer adding onto the `skipUnlessDBFeature` to add in something specific to that function. Thanks!

Reply:
Sorry I wasn't trying to remove the initial `skipUnless`, I think we want both

Reply:
@sarahboyce my bad, misread the suggestion and though it included the removal. I'll tack it back on, thanks!